### DISCUSSION TITLE
Forms don't work on web

### DISCUSSION THREAD

**Discussion by KyleAMathews** [tamagui/tamagui#1885] [General]
### Current Behavior

- `onSubmit` has the event from the button
- `Input` components don't let you pass in any normal props like `name` and `type`

### Expected Behavior

Normal web form behaviors (with great styling!)

### Tamagui Version

```markdown
Normal web form behaviors (with great styling!)
```

### Platform (Web, iOS, Android)

```markdown
Normal web form behaviors (with great styling!)
```

### Reproduction

```markdown
Just build a form
```

**Comment by natew** [comment]
I would like this too, we are just using React Native Web components and wrapping them with Tamagui styles for now, so in order to support this we'd need to make our own components or get React Native / Web to support them.
